.eslintignore

```diff
@@ -0,0 +1,3 @@
+*.test.tsx
+*.ts
+.eslintrc.js
```

Comment:
Also not a _big_ deal, but you can set your editor to trim extra new lines at the end of files and also always make sure there is a new line